common/djangoapps/xblock_django/user_service.py

```diff
@@ -81,7 +81,14 @@ def _convert_django_user_to_xblock_user(self, django_user):
             # need this check because profile is only created through registration flow
             full_name = full_name if full_name is not None and full_name != '' else "{} {}".format(
                 django_user.first_name, django_user.last_name)
+            full_name = full_name.title()
+
+            first_name = ''
+            if full_name.strip() != '':
```

Comment:
@zahid-arbisoft  not necessary but pythonic version looks good :) i.e
`first_name  = '' if full_name.strip() == '' else full_name.split()[0]`

Reply:
Nice catch, I forgot to do it. I have updated the code.